packages/target-decisioning-engine/src/utils.js

```diff
@@ -242,16 +186,6 @@ export function getArtifactFileName(artifactFormat = ARTIFACT_FORMAT_DEFAULT) {
 
   return ARTIFACT_FILENAME[artifactFormat];
 }
-
-/**
- * @param {import("../types/DecisioningConfig").DecisioningConfig} config
- * @return {string}
- */
-export function getGeoLookupPath(config) {
```

Comment:
@shandilya3 @dcottingham Are we sure we can remove this?  This seems to be use by ODD to get the geo location?